comp/api/api/apiimpl/internal/agent/agent.go

```diff
@@ -452,6 +453,14 @@ func secretInfo(w http.ResponseWriter, _ *http.Request, secretResolver secrets.C
 	secretResolver.GetDebugInfo(w)
 }
 
+func secretRefresh(w http.ResponseWriter, _ *http.Request, secretResolver secrets.Component) {
+	if err := secretResolver.Refresh(); err != nil {
+		setJSONError(w, err, 500)
+		return
+	}
+	w.Write([]byte("OK"))
```

Comment:
Random question: Show we print the new secrets so the customer do not need to call `agent secrets` after the refresh?

Reply:
I tried it out, and it does make the CLI interface much better I think.